index.spec.js

```diff
@@ -101,7 +166,7 @@ describe('event-to-promise', function () {
     var error = new Error()
 
     // Node requires at least one error listener.
-    emitter.on('error', function noop () {})
+    emitter.once('error', function noop () {})
```

Comment:
It's called once, after which we have no further use for it. Why would you not use `once` here?

If there was a bug in this library that caused the error event to be fired twice, `once` would expose it, while `on` would silently swallow it.